adam/curriculum/phase2_curriculum.py

```diff
@@ -822,11 +868,12 @@ def prepositional_relation_described_curriculum(
             perception_generator=INTEGRATED_EXPERIMENT_PERCEPTION_GENERATOR,
         )
 
-    ordered_curriculum = [
-        single_object_described_curriculum(num_samples),
-        single_attribute_described_curriculum(num_samples),
-        prepositional_relation_described_curriculum(num_samples),
-    ]
+    ordered_curriculum = [single_object_described_curriculum(num_samples)]
+    if params.boolean("include_attributes", default=True):
+        ordered_curriculum.append(single_attribute_described_curriculum(num_samples))
+    if params.boolean("include_relations", default=True):
+        prepositional_relation_described_curriculum(num_samples)
```

Comment:
Does this need to get appended to the ordered_curriculum? It looks like a no-op.